src/python.cc

```diff
@@ -1443,7 +1460,7 @@ ModelInstanceState::ResponseSendDecoupled(
   ResponseSendMessage* send_message_payload =
       reinterpret_cast<ResponseSendMessage*>(send_message.data_.get());
   std::unique_ptr<PbString> error_message;
-  ScopedDefer _([send_message_payload] {
+  ScopedDefer _([&send_message_payload] {
```

Comment:
Why take reference of a pointer?

Reply:
No particular reason. I'll revert this change.